examples/plot_display_object_visualization.py

```diff
@@ -0,0 +1,91 @@
+"""
+===================================
+Visualizations with Display Objects
```

Comment:
Update the visualization UG to link to this example? (just a link in the example box is enough)